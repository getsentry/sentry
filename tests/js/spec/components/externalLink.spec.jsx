import React from 'react';
import {shallow} from 'sentry-test/enzyme';
import ExternalLink from 'app/components/links/externalLink';

describe('ExternalLink', function() {
  it('renders', function() {
    const wrapper = shallow(<ExternalLink href="https://www.sentry.io/" />);
    expect(wrapper).toMatchSnapshot();
  });
});
