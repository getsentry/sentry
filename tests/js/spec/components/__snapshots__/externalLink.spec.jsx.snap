// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExternalLink renders 1`] = `
<a
  href="https://www.sentry.io/"
  rel="noreferrer noopener"
  target="_blank"
/>
`;
